File: 1. Power.cpp
#include<bits/stdc++.h>
using namespace std;
int Power(int n , int p){
    if(p<=0)
    return 1;
    return n*Power(n,p-1);
}
int main(){
    int p , n ;
    cout<<"Enter number , power"<<endl;
    cin>>n>>p;
    cout<<n<<" raise to "<<p<<" is "<<Power(n,p);
}

File: 1.Power.js
let Power = (n, x) => {
  return !!x ? n * Power(n, x - 1) : 1;
};

const readline = require("readline").createInterface({
  input: process.stdin,
  output: process.stdout,
});

readline.on(`line`, (input) => {
  [n, x] = input.split(" ");
  console.log(Power(Number(n), Number(x)));
  readline.close();
});
